packages/ui-extensions/docs/surfaces/customer-account/reference/examples/apis/settings-access.example.tsx

```diff
@@ -1,24 +1,14 @@
+import '@shopify/ui-extensions/preact';
 import {render} from 'preact';
 import {useEffect, useState} from 'preact/hooks';
 
 export default async () => {
-  render(<App />, document.body);
-}
-
-function App() {
-  const [bannerTitle, setBannerTitle] = useState(
-    shopify.settings.current?.banner_title || '',
-  );
+  render(<Extension />, document.body);
+};
 
-  useEffect(() => {
-    shopify.settings.subscribe(
-      (updatedSettings) => {
-        setBannerTitle(
-          updatedSettings.banner_title,
-        );
-      },
-    );
-  }, []);
+function Extension() {
+  const bannerTitle =
+    shopify.settings.value?.banner_title || '';
```

Comment:
I've tested this change and it works. So I assume similar changes to other examples also work.